content/4_Gold/Stacks.problems.json

```diff
@@ -52,6 +52,18 @@
         "kind": "internal"
       }
     },
+    {
+      "uniqueId": "baltic-12-mobile",
+      "name": "2012 - Mobile",
+      "url": "https://oj.uz/problem/view/BOI12_mobile",
+      "source": "Baltic OI",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Binary Search", "Stack"],
```

Comment:
wait a problem can have different tags if they're in diff modules?

Reply:
I feel like the problems system needs refactoring...like there should be a centralized problems.json file and problems should be referenced only by id within modules

Reply:
i think tags + appearances are different -- tags don't show up on the problem search anymore though 👀

Reply:
isn't there a toggle to show tags in settings 😅 
<img width="616" alt="image" src="https://github.com/cpinitiative/usaco-guide/assets/65467530/9e991b0d-8a1f-4678-970c-f74626ebba10">

Reply:
👀 oops im so outdated -- im not sure which tags take precedence :/